ruby_on_rails/rails_sprinkles/turbo.md

```diff
@@ -20,7 +20,7 @@ This section contains a general overview of topics that you will learn in this l
 Before getting into Turbo, it's important to understand the concept of a Single Page Application, or SPA. A SPA is an
 implementation for web applications that loads only a single web document. Rather than constantly replacing this document with a brand new one on every request, the website dynamically rewrites the current page with new information so that the page loads faster and feels more like a native application. A page refresh never occurs. Think about websites such as Gmail or Facebook where you see a brand-new email or timeline post without needing to refresh your page.
 
-#### Creating a Single Page Application in Rails
+#### Creating a single page application in Rails
```

Comment:
```suggestion
#### Creating a Single Page Application in Rails
```

You can click `commit suggestion` to avoid having to change it locally again :)

Reply:
It didn't work before? I clicked that to the previous 2 suggestions👀

Reply:
it still showed as open to me but now it looks committed! Maybe an async issue between you and me :D